src/test/java/org/kohsuke/github/PullRequestTest.java

```diff
@@ -95,6 +96,17 @@ public void pullRequestReviewComments() throws Exception {
     }
 
     @Test
+    public void testPullRequestReviewRequests() throws Exception {
+        String name = "testPullRequestReviewRequests";
+        GHPullRequest p = getRepository().createPullRequest(name, "test/stable", "master", "## test");
+        System.out.println(p.getUrl());
+        assertTrue(p.getRequestedReviewers().isEmpty());
+
+        GHUser kohsuke2 = gitHub.getUser("kohsuke2");
+        p.requestReviewers(Collections.singletonList(kohsuke2));
+        assertFalse(p.getRequestedReviewers().isEmpty());
+    }
```

Comment:
@farmdawgnation 
Good test for `requestReviewers`, next you'll do a test for `requestTeamReviewers`?

Reply:
Yes, I will.